src/config/dynamic-theme-fixes.config

```diff
@@ -4026,6 +4026,18 @@ CSS
 
 ================================
 
+vice.com
+
+INVERT
+img[src*="1565710323798-article-logo-news.svg"]
+
+CSS
+.titlebar__logo__icon svg path {
+    --darkreader-inline-fill: ${black} !important;
```

Comment:
Remove
```
{
    --darkreader-inline-fill: ${black} !important;
}
```
And change CSS to IGNORE INLINE STYLE

also add `.logo-vice__foreground` For this sites https://www.vice.com/en_us/section/tech

Reply:
Also fixed hamburger menu icon and footer icon. The cards are still white on the main page.